app/assets/stylesheets/govuk_publishing_components/individual_component_support.scss

```diff
@@ -0,0 +1,10 @@
+$govuk-new-link-styles: true;
```

Comment:
This filename should have an underscore since it's used as an include rather than an entry point.

Reply:
Thanks, if it's OK, I've captured this point to work on as part of this Trello card - Trello card: https://trello.com/c/OX7Vgl8B/1698-individual-loading-of-assets-documentation-and-sharing

My thinking here is that the name of `individual_component_support` is currently consistent with the naming of `component_support.scss` and `govuk_frontend_support.scss` in the [same directory](https://github.com/alphagov/govuk_publishing_components/tree/main/app/assets/stylesheets/govuk_publishing_components), but we may want to change these file names as well.

Reply:
I think @kevindew's point here is that this is different from` component_support.scss` and `govuk_frontend_support.scss`, which are both intended to be included in other apps, whereas `individual_component_support.scss` is only included in other stylesheets in this gem and isn't intended to be used outside the gem.

Reply:
Done 👍